PowerEditor/src/NppDarkMode.cpp

```diff
@@ -349,12 +349,16 @@ namespace NppDarkMode
 		return opt;
 	}
 
+	static bool _isAtLeastWindows10 = false;
+
 	void initDarkMode()
 	{
 		_options = configuredOptions();
 
 		initExperimentalDarkMode();
 		setDarkMode(_options.enable, true);
+
+		_isAtLeastWindows10 = NppDarkMode::isWindows10();
```

Comment:
@donho 
I am not sure why there is issue in Windows 8.1 since custom draw should work from Windows Vista and later.
I have not tested it on Windows 8.1 but I tried Windows 7 and it works.
But since `SetWindowTheme(hwnd, L"DarkMode_Explorer", nullptr);` works only on Windows 10 and later, this might caused issue with older OS, therefore this PR.